src/duckdb.js

```diff
@@ -166,7 +169,7 @@ async function insertFile(database, name, file, options) {
     const buffer = await file.arrayBuffer();
     await database.registerFileBuffer(file.name, new Uint8Array(buffer));
   } else {
-    await database.registerFileURL(file.name, url);
+    await database.registerFileURL(file.name, url, duckdb.DuckDBDataProtocol.HTTP);
```

Comment:
Oops, unfortunately the `duckdb` here refers to the wrong thing:

https://github.com/observablehq/stdlib/blob/dc5965e571c375688de21d8ef53fd5b9a223d00b/src/dependencies.js#L23

I’ll fix and then merge.